src/lib/tabs/tab-header.spec.ts

```diff
@@ -29,7 +31,9 @@ describe('MdTabHeader', () => {
         SimpleTabHeaderApp,
       ],
       providers: [
-        {provide: Dir, useFactory: () => { return {value: dir}; }},
+        {provide: Dir, useFactory: () => {
+          return {value: dir,  dirChange: dirChange.asObservable()};
+        }},
```

Comment:
Done. I went with a spy, instead of testing the position itself, because knowing when the position is final is hard since we're running things outside of Angular and we're using CSS transitions to animate it. I had something working, but it was using `transitionend` which may end up making the test flaky.